glean-core/src/histogram.rs

```diff
@@ -36,10 +36,10 @@ fn exponential_range(min: u32, max: u32, bucket_count: usize) -> Vec<u32> {
     ranges.push(current);
 
     for i in 2..bucket_count {
-        let log_current = f64::from(current).ln();
+        let log_current = (current as f64).ln();
```

Comment:
This will lose precision if `min` and `max` are larger than 53 bits.  But for exponential bucketing, those lower bits don't matter anyway, so probably not an issue in practice, but something to flag.  If we *really* wanted to be safe, I'd cap min and max at 2 ^ 53 - 1 at build (glean_parser) time.

Reply:
Quite honestly, we're already loosing that if we ever have anything above 2^53 in the output data anyway (yey JSON being basically javascript with no real integers)